src/MasterSite.php

```diff
@@ -1248,16 +1248,27 @@ public function add_image_attachment_fields_to_save( $post, $attachment ) {
 	}
 
 	/**
-	 * Override the Gutenberg core/image block render method output, To add credit field in it's caption text.
+	 * Override the Gutenberg core/image block render method output, To add credit field in it's caption text & image alt text as title.
 	 *
 	 * @param array  $attributes    Attributes of the Gutenberg core/image block.
 	 * @param string $content The image element HTML.
 	 *
-	 * @return string HTML content of image element with credit field in caption.
+	 * @return string HTML content of image element with credit field in caption and alt text in image title.
 	 */
 	public function p4_core_image_block_render( $attributes, $content ) {
-		$image_id = isset( $attributes['id'] ) ? trim( str_replace( 'attachment_', '', $attributes['id'] ) ) : '';
-		$credit   = $image_id ? get_post_meta( $image_id, self::CREDIT_META_FIELD, true ) : '';
+		$image_id      = isset( $attributes['id'] ) ? trim( str_replace( 'attachment_', '', $attributes['id'] ) ) : '';
+		$img_post_meta = $image_id ? get_post_meta( $image_id ) : [];
+		if ( ! $img_post_meta ) {
+			return $content;
+		}
+
+		$credit   = $img_post_meta[ self::CREDIT_META_FIELD ][0] ?? '';
+		$alt_text = $img_post_meta['_wp_attachment_image_alt'][0] ?? '';
+
+		if ( $alt_text ) {
+			$content = str_replace( 'alt=', 'title="' . esc_attr( $alt_text ) . '" alt=', $content );
+		}
+
 		if ( empty( $image_id ) || empty( $credit ) ) {
```

Comment:
With the new check you added earlier, it's now impossible that it gets here and `empty( $image_id )` would be true, so I guess we can remove that part of the condition.